### ISSUE TITLE
Deep Optimization of Parametric IIR Filters for Audio Equalization

### ISSUE THREAD

**Issue by thereisatablehere** [thereisatablehere/CapstoneResources#3]
### Summary
This paper describes a novel Deep Learning method for the design of IIR parametric filters for automatic audio equalization. A simple and effective neural architecture, named BiasNet, is proposed to determine the IIR equalizer parameters. An output denormalization technique is used to obtain accurate tuning of the IIR filters center frequency, quality factor and gain. All layers involved in the proposed method are shown to be differentiable, allowing backpropagation to optimize the network weights and achieve, after a number of training iterations, the optimal output. The parameters are optimized with respect to a loss function based on a spectral distance between the measured and desired magnitude response, and a regularization term used to achieve a spatialization of the acoustc scene. Two scenarios with different characteristics were considered for the experimental evaluation: a room and a car cabin. The performance of the proposed method improves over the baseline techniques and achieves an almost flat band. Moreover IIR filters provide a consistently lower computational cost during runtime with respect to FIR filters.

### Key Points
- BiasNet
- IIR Filters
- Gradient-Based Optimization

### Citation
G. Pepe, L. Gabrielli, S. Squartini, C. Tripodi, and N. Strozzi, "Deep optimization of parametric IIR filters for audio equalization," _arXiv_, 2021. [Online]. Available: https://arxiv.org/abs/2110.02077.

### Repo link
N/ A
